lib/src/components/help_options_list.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+
+import 'package:help_ukraine_widget/src/components/help_option_button.dart';
+
+/// It's a class that represents an item in the list of actions
+class HelpOptionsListItem {
```

Comment:
What's the purpose of splitting this out into a separate structure?
It completely duplicates the fields of `HelpOptionButton`, and if we need to make changes to it -- we need to reflect them here.

Reply:
I just like using BottomNavigationBar as a reference, but items in this widget have a lot of processing, so in our case it is realy useless.